scripts/exregional_run_analysis.sh

```diff
@@ -704,6 +829,41 @@ done
 if [ ${gsi_type} == "OBSERVER" ]; then
   cp_vrfy *diag* ${observer_nwges_dir}/.
 fi
+
+#
+#-----------------------------------------------------------------------
+#
+# cycling radiance bias corretion files
+#
+#-----------------------------------------------------------------------
+
+if [ ${DO_RADDA} == "TRUE" ]; then
+
+  if [ ${cycle_type} == "spinup" ]; then
+     spinup_or_prod_rrfs=spinup
+  else
+     spinup_or_prod_rrfs=prod
+  fi
+
+
+  tar -cvzf rrfs.${spinup_or_prod_rrfs}.${YYYYMMDDHH}_radstat *diag*.${YYYYMMDDHH}
+
+  cp_vrfy ./satbias_out ${SATBIAS_DIR}/rrfs.${spinup_or_prod_rrfs}.${YYYYMMDDHH}_satbias
+  cp_vrfy ./satbias_pc.out ${SATBIAS_DIR}/rrfs.${spinup_or_prod_rrfs}.${YYYYMMDDHH}_satbias_pc
+  cp_vrfy ./rrfs.${spinup_or_prod_rrfs}.${YYYYMMDDHH}_radstat  ${SATBIAS_DIR}/rrfs.${spinup_or_prod_rrfs}.${YYYYMMDDHH}_radstat
+
+
+  if [ ${cycle_type} == "prod" ]; then
+     if [ ${HH} -eq '00' ] || [ ${HH} -eq '01' ] || [ ${HH} -eq '02' ] || [ ${HH} -eq '09' ] || [ ${HH} -eq '10' ] || [ ${HH} -eq '11' ] ||[ ${HH} -eq '12' ] || [ ${HH} -eq '13' ] || [ ${HH} -eq '14' ] || [ ${HH} -eq '21' ] || [ ${HH} -eq '22'] || [ ${HH} -eq '23' ]; then
+      ln -s ${SATBIAS_DIR}/rrfs.${spinup_or_prod_rrfs}.${YYYYMMDDHH}_satbias ${SATBIAS_DIR}/rrfs.spinup.${YYYYMMDDHH}_satbias
+      ln -s ${SATBIAS_DIR}/rrfs.${spinup_or_prod_rrfs}.${YYYYMMDDHH}_satbias_pc ${SATBIAS_DIR}/rrfs.spinup.${YYYYMMDDHH}_satbias_pc
+      ln -s ${SATBIAS_DIR}/rrfs.${spinup_or_prod_rrfs}.${YYYYMMDDHH}_radstat ${SATBIAS_DIR}/rrfs.spinup.${YYYYMMDDHH}_radstat
+     fi
```

Comment:
Why do we need to make those links?

Reply:
This is for a rare condition just in case we don't have satbias files from the spinup cycles but would like to use the latest available satbias files especially for the 09z (and 21z) full cycle.  I removed these lines for now since they included some fixed cycle hours.  We can have a better way for this if needed in the future.  I will make a new PR based on a new branch to replace this PR due to some conflicts.  Thanks for your very useful review and comments!